client/app/components/Button.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
```

Comment:
I think @pmdarrow was using some kind of UI library. Maybe we can find a button and a Container components there so we don't need to create our owns